package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "babel-core": "^6.9.0",
+    "babel-plugin-transform-amd-system-wrapper": "^0.1.0",
```

Comment:
Ok, that seems all the uses are then just the `getDeferredImports` function (which takes a source string, and returns the arguments of all `System.import('x')` calls in that source.), and the ESM remap function (which takes an esm source, and returns a new esm source with the imports remapped by the given map function).

Both of these are completely separate things though so can be a separate PR at some point in future. There is no rush to upgrade these.

I think just getting this transformer ready for 0.16 would be ideal fixing up the orderings.